### PR TITLE
Adding Changes 

### PR DESCRIPTION
**Pull request by melashu**
## New files 
- Add html file
- Add CSS file 
- Update gitignore file

### CODE REVIEW

File: a.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.header {
+  color: brown;
+  font-size: 2px;
+}
```
Comment (1GORDON) [inline a.css]: Great job @melashu. 👏

- [ ] Please try to increase the font size of your header class. as `2px` this is too small to be visible in the browser

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @melashu  

Good job so far!

## Top Highlights 
- Good feature branch name ✔️ 
- Good commit messages  ✔️ 
- professional Readme ✔️
- Linters are passing ✔️
- Professional Pull request and its description ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] In the future try to work off of the main or master branch instead of other branches.

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread `1GORDON` if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: a.css (hunk 1 above)
Comment (melashu) [reply a.css]: Thank you for your constructive comment.

**Review by danielufeli — APPROVED** [review]
Hi @melashu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Good use of GitHub flow ✔️
- No Linter Errors ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**